src/api/routes/gem2s.js

```diff
@@ -0,0 +1,54 @@
+const AWSXRay = require('aws-xray-sdk');
+const { createGem2SPipeline } = require('../general-services/pipeline-manage');
+const ExperimentService = require('../route-services/experiment');
+const getBackendStatus = require('../general-services/backend-status');
+const pipelineResponse = require('../route-services/pipeline-response');
+const parseSNSMessage = require('../../utils/parse-sns-message');
+const logger = require('../../utils/logging');
+
+module.exports = {
+  'gem2s#get': (req, res, next) => {
+    getBackendStatus(req.params.experimentId)
+      .then((data) => res.json(data))
+      .catch(next);
+  },
+
+  'gem2s#create': (req, res, next) => {
+    createGem2SPipeline(req.params.experimentId)
+      .then((data) => {
+        const experimentService = new ExperimentService();
+        experimentService.savePipelineHandle(req.params.experimentId, data)
+          .then(() => res.json(data));
+      })
+      .catch(next);
+  },
+
+  'gem2s#response': async (req, res) => {
```

Comment:
Do we need this? I don't see the yaml file where this route is created, but could we send this route to `pipelines#response`? Same question for `gem2s#get`

Reply:
no, because they are going to be different. This will be used by later PR from martin and/or agi.